src/components/FlowPlayer/FlowPlayer.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('<FlowPlayer />', () => {
 			<FlowPlayer className={customClass} {...MOCK_FLOW_PLAYER_PROPS} src={null} />
 		);
 
-		expect(FlowPlayerComponent.find('.c-cut-overlay')).toHaveLength(1);
+		expect(FlowPlayerComponent.find('.c-cut-overlay')).toHaveLength(0);
```

Comment:
How is die test ooit gelukt aan mijn kant? :eyes:

Reply:
Geen idee 😄